### ISSUE TITLE
Should automatically close the issue, if the PR is merged!

### ISSUE THREAD

**Issue by jain-rishabh-21** [jain-rishabh-21/DoctorVerse#53]
## Describe the bug/feature
Should automatically close the issue, if the PR is merged!

**Comment by Soumya1219** [comment]
Kindly assign me with this issue :) 
I will fix it. 🔧

**Comment by apeksha235** [comment]
Assigned it to you @Soumya1219

**Comment by jain-rishabh-21** [comment]
<img width="1059" alt="image" src="https://github.com/jain-rishabh-21/DoctorVerse/assets/35010022/412c94ca-ac76-475f-820b-c1bb33dba5b9">
Getting this error! After merging your PR! Can you have a look!

**Comment by Priyank-Vaidya** [comment]
Hello @apeksha235, Can you assign me this issue, I can help to fix it.

**Comment by Soumya1219** [comment]
Regarding the issue to be automatically closed once the PR is merged!
You said that there was an error in the code can you let me know actually
what was the error?
And what is mean by revert?
How should I make changes?

**Comment by apeksha235** [comment]
hey @Soumya1219 , I think it's a formatting issue, can you check it once. 
- you can make changes again and create a new pull request. 
- make sure it's done in a day or two, or we will have to assign it to someone else
